zk/tests/compose/cert.pem

```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----
+MIIDfzCCAmegAwIBAgIEczS1BjANBgkqhkiG9w0BAQsFADBvMRAwDgYDVQQGEwdV
```

Comment:
Can we use different certs for `cert.pem` and `ca_cert.pem`?

Reply:
I originally generated the `pem` files from a simple generated `keystore` and `truststore`, and it resulted in the same `cert` and `ca_cert` files.

Reply:
Not sure I understand. You've create a key pair + certificate for the server to use. This is the cert used by the server in its keystore, and the cert that the client needs to define as its truststore (i.e the ca_cert file). This on its own is enough for using SSL, the client simply gives the certificate it trusts, and the server uses that same certificate in its keystore (the server also has the private key used to proove that he can be trusted). This is cert A.

On top of that there is client authentication; where the client uses a certificat + private key in its keystore. To support this the server needs to add this client certificate in its truststore. This is cert B.

In practice cert A and cert B will never be the same, so having a more realistic use case here seems important. By using the same cert as the client and the server cert, we might assume the integration works correctly while it "swaps" the two, we would have no way to tell.